Source/FieldSolver/PicsarHybridSpectralSolver/PicsarHybridSpectralSolver.cpp

```diff
@@ -5,10 +5,10 @@
 
 using namespace amrex;
 
-constexpr int WarpX::FFTData::N;
+constexpr int FFTData::N;
 
 namespace {
-static std::unique_ptr<WarpX::FFTData> nullfftdata; // This for process with nz_fft=0
+static std::unique_ptr<FFTData> nullfftdata; // This for process with nz_fft=0
```

Comment:
`FFTData` used to be a member of the `WarpX` class. This was not really needed, and is no longer the case now.